Tests/Marketplace/marketplace_services.py

```diff
@@ -949,6 +1019,97 @@ def prepare_release_notes(self, index_folder_path, build_number):
         finally:
             return task_status, not_updated_build
 
+    def prepare_release_notes(self, index_folder_path, build_number):
```

Comment:
What happens if there is more than one version merged to master in a single pack?

Until now it could not have happened because each commit was uploading for itself, but now that it's scheduled you could miss release notes or squash them.

Reply:
see the implementation of `get_release_notes_lines` func